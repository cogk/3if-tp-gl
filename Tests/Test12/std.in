2
2
01/01/2010/123456/1
