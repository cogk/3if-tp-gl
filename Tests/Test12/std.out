* Bienvenue dans AirWatcher 
| Utilisateur: (anonyme)       
* Menu Principal 

   1. Contribution…
   2. Analyse…
   3. Sources de données…
   4. Administration…
   5. Super Admin…
   6. Quitter AirWatcher

Entrez ici votre sélection : 
* Menu - Analyse 

   1. Filtrage géographique…
   2. Filtrage temporel…
   3. Lancer le calcul
   4. Retour au menu principal

Entrez ici votre sélection : 
* Définir une période d'analyse 
Date de début (jj/mm/aaaa) : Date de fin   (jj/mm/aaaa) : Date invalide.
* Menu Principal 

   1. Contribution…
   2. Analyse…
   3. Sources de données…
   4. Administration…
   5. Super Admin…
   6. Quitter AirWatcher

Entrez ici votre sélection : 
Entrée invalide, veuillez entrer un entier entre 1 et 6.
Vous n'avez pas choisi de menu.
